src/gui/MainWindow.cpp

```diff
@@ -40,6 +40,12 @@ MainWindow::MainWindow(GladeSearchpath* gladeSearchPath, Control* control)
 	this->toolbarWidgets = new GtkWidget*[TOOLBAR_DEFINITIONS_LEN];
 	this->toolbarSelectMenu = new MainWindowToolbarMenu(this);
 
+	
+	
+	GtkOverlay *overlay = GTK_OVERLAY (get("mainOverlay"));
+	this->floatingToolbox = new FloatingToolbox (this, overlay);  
```

Comment:
No need to make `floatingToolbox` a pointer if it's only going to be allocated in the constructor and deallocated in the destructor.

Reply:
I started that way... But wanted to avoid a separate initialization function.  Critical during testing but might only be a preference now. Leaving for now at least.